lib/CppInterOp/CppInterOp.cpp

```diff
@@ -3191,6 +3214,19 @@ TInterp_t CreateInterpreter(const std::vector<const char*>& Args /*={}*/,
     llvm::cl::ParseCommandLineOptions(NumArgs + 1, Args.get());
   }
 
+  if (GpuArgs.empty()) {
```

Comment:
Why is this irrelevant for gpu?

Reply:
This crashes with the `--cuda` flag. Atleast on my machine.

Reply:
Why?

Reply:
Apparently the crash is a specific problem with my machine. The CI is green for Ubuntu.
But fails on WASM.